sdk/spring/spring-cloud-azure-supported-spring.json

```diff
@@ -85,15 +85,15 @@
     "snapshot" : false,
     "supportStatus" : "SUPPORTED",
     "spring-boot-version" : "2.5.14",
-    "spring-cloud-version" : "2020.0.5"
+    "spring-cloud-version" : "2020.0.6"
   },
   {
     "current" : false,
     "releaseStatus" : "GENERAL_AVAILABILITY",
     "snapshot" : false,
     "supportStatus" : "END_OF_LIFE",
     "spring-boot-version" : "2.4.13",
-    "spring-cloud-version" : "2020.0.5"
+    "spring-cloud-version" : "2020.0.6"
```

Comment:
Why do we update this version for 2.4.13?

Reply:
I guess he is thinking that `spring-cloud:2020.0` is compatible with `spring-boot:2.4` and `spring-boot:2.5`.

I think it's OK to keep it updated even though `spring-boot:2.4` is `END_OF_LIFE`.

Reply:
This file should reflect our support status and the result of compatibility tests against different Spring Boot and Spring Cloud versions. So if a combo of these two versions has been tested and is the latest, we mark them as `SUPPORTED`, but if it is `EOL`, I don't think we should update the combo anymore.